### PR TITLE
base.html: Improve navbar

### PR DESCRIPTION
**Pull request by sagar-kalra**
Improve navbar and add navstyle.css for styling

Fixes  https://github.com/pclubuiet/website/issues/2 .

### CODE REVIEW

**Comment by divyam3897** [comment]
@sagar-kalra still on it?

**Comment by sagar-kalra** [comment]
@divyam3897 yeah, I ll complete it.

**Comment by sagar-kalra** [comment]
Ignore this commit

**Comment by sagar-kalra** [comment]
@divyam3897 I have designed the navbar, but i am not able to link about us.

File: templates/base.html [hunk 1]
```diff
@@ -29,35 +30,38 @@
...
   <body  {% block body_style %} {% endblock %}>
 <!--CSS Loader -->
 <div class="loader-wrapper">
   <div class="loader">
-  	<div class="loader-rect1"></div>
-  	<div class="loader-rect2"></div>
-  	<div class="loader-rect3"></div>
-  	<div class="loader-rect4"></div>
-  	<div class="loader-rect5"></div>
+    <div class="loader-rect1"></div>
```
Comment (divyam3897) [inline templates/base.html]: why these changes?

File: staticFiles/home/base.css [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+  min-height: 83vh; /* These two lines are counted as one :-)       */
+
+  display: flex;
+  align-items: center;
+}
+
+.jumbotron h1{
+	font-size: 6.2vw !important;
+}
+
```
Comment (divyam3897) [inline staticFiles/home/base.css]: why these empty lines?

File: templates/base.html (hunk 1 above)
Comment (divyam3897) [reply templates/base.html]: Remove these changes!!
